functions/venue.js

```diff
@@ -102,6 +102,10 @@ const createVenueData = (data, context) => {
     }
   }
 
+  if (data.mapBackgroundImageUrl) {
+    venueData.mapBackgroundImageUrl = data.mapBackgroundImageUrl;
```

Comment:
I think the way it was implemented differs from the one suggested here.

![image](https://user-images.githubusercontent.com/63194656/104582540-ad5be380-5668-11eb-9a91-c16682879ec3.png)

Reply:
@mike-lvov Yes it differs, since camp is going to be removed in the future. I decided to make it a conditional check, that way in the future when a new venue is added and mapBackgroundImagUrl is provided you won't have to go in the cases and put it in there again.

Reply:
> Can we move it here?
> 


Yes, will do.